Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -341,6 +342,7 @@ while ($currentDiff -gt 0) {
         $ModuleAllowList.add("SCCOLE2.dll")
         $ModuleAllowList.add("SCCSD.dll")
         $ModuleAllowList.add("SCCXT.dll")
+        $ModuleAllowList.add("vshtml.dll")
```

Comment:
looks like there are a lot of possible dlls that we can load from that location (`C:\Program Files\Microsoft\Exchange Server\V15\FIP-FS\Bin\TE_v.8.5.3.0`) might be best to avoid this happening again moving forward is to account for this somehow.

Reply:
Let me know if you want it to be apart of this PR or not.

Reply:
I think we can check on FIP-FS if the company is Oracle, as we have several dlls there from oracle 

![image](https://github.com/microsoft/CSS-Exchange/assets/103440830/9eb632ea-29f8-4750-97c0-3e69832737a1)

![image](https://github.com/microsoft/CSS-Exchange/assets/103440830/582eee73-d591-4a80-a5a3-2f39febe3c4a)